README.md

```diff
@@ -31,6 +32,19 @@ What's included in this template:
  * Pytest example(s).
  * Githooks.
 
+# Quickstart Guide
+
+Add here, streamlined instructions on how to get running and using the code as swiftly as possible. This shouldn't
+attempt to cover all OS's and/or build variations - just the canonical. Since users are most likely viewing this README
+from GitHub.com a repo context might be best where instructions look like those below. Alternatively, if this package is
+distributed on PyPi, perhaps just ``pip install <package-name>``, followed by quick user instructions will suffice.
+
+  * ``git clone <git-url>``
```

Comment:
Could add base-template url so that ``project_setup.py`` does its thing...
Same goes for ``env-name`` also, perhaps.

Reply:
Good idea.